src/rotas/usuario/usuarioRotas.py

```diff
@@ -51,13 +51,29 @@ class Token(BaseModel):
     tags=["Usuário"],
 )
 
+## Para mais detalhes sobre a autenticação, confira o documento
+## [docs/auth](docs/auth.md).
 
 tokenAcesso: OAuth2PasswordBearer = OAuth2PasswordBearer(
     tokenUrl=config.ROOT_PATH + "/usuarios/login"
 )
+"""
+Portador de token de acesso.
+
+Se especificado como dependência de uma rota, torna-a autenticada.
+"""
 
 
 def getUsuarioAutenticado(token: Annotated[str, Depends(tokenAcesso)]):
+    """
+    Recupera o token de acesso fornecido na solicitação e retorna um objeto usuário cujo
+    token lhe pertence.
+    
+    Caso o token seja inválido ou não exista, retorna uma resposta HTTP 401.
+
+    Pode ser especificada como dependência para obter o usuário autenticado em rotas
+    que precisam de autorização.
```

Comment:
Args:
        token (str): Token de acesso fornecido pelo cliente na solicitação. 
        Esse token é utilizado para identificar e autenticar o usuário.

    Returns:
        Usuario: Objeto contendo as informações do usuário autenticado.

    Raises:
        HTTPException: Retorna uma resposta HTTP 401 se o token for inválido, 
        estiver ausente ou não corresponder a um usuário válido.